modules/Bio/EnsEMBL/Production/DBSQL/BulkFetcher.pm

```diff
@@ -390,6 +415,24 @@ sub get_transcripts {
     return $transcript_hash;
 } ## end sub get_transcripts
 
+
+sub set_cds {
+
+  my ($self , $exons_list, $transcript, $transcript_id ) = @_;
+
+  $log->debug("Getting CDS Data"); 
```

Comment:
Could you remove this line, please? Or update the log to be more "detailed".

Reply:
ok i will add detailed desc

Reply:
we need to explicitly mention in EBeyeFormatter.pm module for cds data changes to reflect the in xml.  @Kieron Taylor was concern about the json data.

Reply:
Ok so we don't change EBEYE format just yet. We'll ask Ebi search team about it, thanks.